src/ui/public/documentation_links/documentation_links.ts

```diff
@@ -85,7 +85,10 @@ export const documentationLinks = {
     loadingData: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/tutorial-load-dataset.html`,
     introduction: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/index-patterns.html`,
   },
-  kibana: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/index.html`,
+  kibana: {
+    visualize: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/visualize.html`,
```

Comment:
Bonus assignment: At this point we could (should?) add a `KIBANA_DOCS` constant.

Reply:
Yeah, we should not :-) Read the documentation on the top of the file. This is a super automagic file and if we manipulate those constants we need to fix some external tools around documentation :D